README.rst

```diff
@@ -337,12 +337,39 @@ query returning a dictionary where there are multiple results under each key. No
 ~~~~~~~~~~
 Handles any SQL that is not a select. This is primarily, but not limited to, ``insert``, ``update``, and ``delete``.
 
+
 .. code-block:: python
 
     @sqlupdate()
-    def insert_items(item_dict)
+    def insert_items(item_dict):
         return QueryData("INSERT INTO", template_params={'in__item_id':item_id_list})
 
+You can yield multiple QueryData objects. This is done in a transaction and it can be helpful for data integrity or just
+a nice clean way to run a set of queries
```

Comment:
I think you should change this to "run a set of updates" since I typically see queries as SELECT. But that may just be me. Also, period at the end :)